backend/open_webui/models/users.py

```
@@ -300,5 +339,105 @@ def get_user_api_key_by_id(self, id: str) -> Optional[str]:
+            # Generate all date strings first to ensure no gaps
+            date_strings = []
+            dates_timestamps = []
+            for day in range(days):
+                # Calculate day start (midnight) for each day in the past
+                day_start = today_midnight - (day * 24 * 60 * 60)
+                date_str = time.strftime("%Y-%m-%d", time.localtime(day_start))
+                date_strings.append(date_str)
+                dates_timestamps.append(day_start)
+
+            # Sort date strings to ensure chronological order
+            date_pairs = sorted(zip(date_strings, dates_timestamps))
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)